src/psyclone/tests/domain/lfric/kernel/lfric_kernel_metadata_test.py

```diff
@@ -99,104 +112,79 @@ def test_init_args_error():
     '''
     with pytest.raises(ValueError) as info:
         _ = LFRicKernelMetadata(operates_on="invalid")
-    assert ("The operates_on metadata should be a recognised iteration "
-            "space (one of ['cell_column', 'domain', 'dof']) but found "
-            "'invalid'." in str(info.value))
+    print(str(info.value))
```

Comment:
Please rm this line.

Reply:
Oops. And there was one more print in this file too.